src/main/scala/com/github/tkawachi/doctest/DoctestPlugin.scala

```diff
@@ -83,16 +86,34 @@ object DoctestPlugin extends AutoPlugin {
     doctestMarkdownEnabled := (doctestMarkdownEnabled ?? false).value,
     doctestMarkdownPathFinder := baseDirectory.value * "*.md",
     testFrameworks += new TestFramework("utest.runner.Framework"),
+    doctestIgnoreRegex := None,
     doctestGenTests := {
       (managedSourceDirectories in Test).value.headOption match {
         case None =>
-          streams.value.log.warn("managedSourceDirectories in Test is empty. Failed to generate tests")
-          Seq()
+          streams.value.log.warn("DocTest: managedSourceDirectories in Test is empty. Failed to generate tests")
+          Seq.empty
         case Some(testDir) =>
           val testGen = TestGenResolver.resolve(doctestTestFramework.value, Classpaths.managedJars(Test, classpathTypes.value, update.value), scalaVersion.value)
 
+          val sourceFiles = (unmanagedSources in Compile).value ++ (managedSources in Compile).value
+          val log = streams.value.log
+          log.debug(s"DocTest: Applying ignore pattern [${doctestIgnoreRegex.value}] to exclude matching sources...")
+          val filteredSourceFiles =
+            doctestIgnoreRegex.value.fold(sourceFiles) { regex =>
+              val IgnoreRegex = regex.r
+              sourceFiles.filterNot { f =>
+                FilenameUtils.normalize(f.getCanonicalPath) match {
```

Comment:
`FilenameUtils.normalize()` doesn't normalize path separators according to the document.

> https://commons.apache.org/proper/commons-io/javadocs/api-2.5/org/apache/commons/io/FilenameUtils.html#normalize(java.lang.String)
>
> The output will contain separators in the format of the system.

`normalize(String, boolean)` can be used to normalize path separators. I think it's better to normalize to `/` because many developers use Unix.

> https://commons.apache.org/proper/commons-io/javadocs/api-2.5/org/apache/commons/io/FilenameUtils.html#normalize(java.lang.String,%20boolean)
>
> The input may contain separators in either Unix or Windows format. The output will contain separators in the format specified.

Reply:
sure, will update shortly